src/systempolicy/observability.go

```diff
@@ -103,7 +106,14 @@ func GetSystemObsData(wpfs types.WorkloadProcessFileSet) (opb.Response, error) {
 	return opbSysObsResponse, nil
 }
 
-func ClearSysDb(wpfs types.WorkloadProcessFileSet, duration int64) error {
-	err := libs.ClearWPFSDb(CfgDB, wpfs, duration)
+func ClearSysDb(wpfs types.WorkloadProcessFileSet, durationStr string) error {
+	if durationStr == "0" {
+		return errors.New("not a valid duration")
+	}
```

Comment:
What is the point of this check? 
If the format is invalid then time.ParseDuration() will return an err.
```suggestion
```

Reply:
Yes Rahul. But when we input 0, this is considered as a valid input and the output is given as 0 seconds.
That is why I specifically included this check.